WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/AddProductName/AddProductNameWithAIView.swift

```diff
@@ -0,0 +1,225 @@
+
+                    // Subtitle label.
+                    Text(Localization.subtitle)
+                        .foregroundColor(Color(.secondaryLabel))
+                        .bodyStyle()
+                }
+                .padding(.bottom, Layout.titleBlockBottomPadding)
+
+                VStack(alignment: .leading, spacing: Layout.titleBlockSpacing) {
+                    VStack(alignment: .leading, spacing: Layout.editorBlockSpacing) {
+                        Text(Localization.productName)
+                            .foregroundColor(Color(.label))
+                            .subheadlineStyle()
+
+                        ZStack(alignment: .topLeading) {
+                            VStack(spacing: 0) {
+                                TextEditor(text: $viewModel.productNameContent)
+                                    .bodyStyle()
+                                    .foregroundColor(.secondary)
+                                    .padding(insets: Layout.messageContentInsets)
+                                    .frame(minHeight: Layout.minimumEditorHeight, maxHeight: .infinity)
+                                    .focused($editorIsFocused)
+
+                                Divider()
+                                    .frame(height: Layout.dividerHeight)
+                                    .foregroundColor(Color(.separator))
+
+                                suggestNameField
+                                    .padding(insets: Layout.suggestButtonInsets)
+                            }
+                            .overlay(
+                                RoundedRectangle(cornerRadius: Layout.cornerRadius).stroke(Color(.brand))
+                            )
+
+                            // Placeholder text
+                            placeholderText
+                        }
+                    }
+
+                    usePackagePhoto
+                }
+            }
+            .padding(insets: Layout.insets)
+        }
+        .safeAreaInset(edge: .bottom) {
+            VStack {
+                // CTA to continue to next screen.
+                continueButton
+                    .padding()
+            }
+            .background(Color(uiColor: .systemBackground))
+        }
+    }
+}
+
+private extension AddProductNameWithAIView {
+    var suggestNameField: some View {
+        HStack {
+            // Suggest a name
+            Button {
+                viewModel.didTapSuggestName()
+            } label: {
+                HStack(alignment: .top) {
+                    Image(uiImage: .sparklesImage)
+                        .renderingMode(.template)
+                        .resizable()
+                        .foregroundColor(Color(.brand))
+                        .frame(width: Layout.sparkleIconSize * scale, height: Layout.sparkleIconSize * scale)
+
+                    Text(Localization.suggestName)
+                        .foregroundColor(Color(.brand))
+                        .bodyStyle()
```

Comment:
🔍 The text should be semibold according to the design.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/10723/commits/887d2da39a473a3903d91bc9f73f663f419eaa11